rotkehlchen/assets/asset.py

```diff
@@ -402,6 +402,61 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     strethaddress_to_identifier('0xBB0E17EF65F82Ab018d8EDd776e8DD940327B28b'): 'AXS',
 }
 
+WORLD_TO_UPHOLD = {
+    'BTC': 'BTC',
+    strethaddress_to_identifier('0xc4f33F15ed2F2C5F8D5B542dD30518A50D9f843f'): 'BTC0',
+    'AAVE': 'AAVE',
```

Comment:
This and all other ethereum tokens would need to have their rotki identifier here on the left side.

There is two ways to go about this:

1. As the other parts of this file show, use `strethaddress_to_identifier'. For example here `strethaddress_to_identifier('0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9')`.
2. If the asset is in the assets/constants.py: https://github.com/rotki/rotki/blob/develop/rotkehlchen/constants/assets.py import it and use its identifier: `A_AAVE.identifier`. Do not add any new assets in the constants for this. Only use one if you want if it's already there.

Reply:
@LefterisJP importing A_BTC causes a circular dependency with the Asset class I think, I used manual inspection to identify the identifiers on my list and commented ones that did not have constants defined.